src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-events.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: 'OpenTelemetry in the UI: Events'
+tags:
+  - Integrations
+  - Open source telemetry integrations
+  - OpenTelemetry
+metaDescription: You can query experimental OpenTelemetry events in New Relic.
+---
+
+import otelEventsQueryScreenshot from 'images/otel-events.png'
+
+
+<Callout title="Limited preview">
+  If you sign up for the limited preview of the Events API, you can query OpenTelemetry events in New Relic. Let us know in our [CNCF Slack channel, #otel-newrelic](https://cloud-native.slack.com/archives/C024DRQ63UP) if you'd like to participate.
+</Callout>
+
+OpenTelemetry recently introduced the experimental [Events API](https://opentelemetry.io/docs/reference/specification/logs/event-api/) in its specification. This API allows you to create an `EventLogger`, which can emit `Events` as `LogRecords` containing two special attributes:
+* `event.domain`: The domain of the emitted events. It needs to be be set during the creation of the `EventLogger`.
+* `event.name`: The event name. This attribute is what New Relic uses to map an OpenTelemetry event into a [New Relic eventType](https://docs.newrelic.com/docs/data-apis/custom-data/custom-events/data-requirements-limits-custom-event-data/#general)
```

Comment:
This isn't clear to me. what does it mean to map an OTel event to a `eventType'? `eventType` is an attribute on custom events I think, so this isn't that clear. is this saying that this is creating a custom event and setting the eventType to whatever event.name is? We should maybe make that a bit clearer, maybe saying something like 'This feature brings in events via our Event API and brings them in as custom events with the name set with the eventType attribute.' Or something like that.